sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -6003,6 +6003,19 @@ object SQLConf {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("3GB")
 
+  val LOCAL_RELATION_BATCH_OF_CHUNKS_SIZE_BYTES =
+    buildConf(SqlApiConfHelper.LOCAL_RELATION_BATCH_OF_CHUNKS_SIZE_BYTES_KEY)
+      .internal()
+      .doc("The batch size in bytes for uploading chunks of local relations to the server. " +
+        "The client collects multiple chunks into a single batch until this limit is reached, " +
+        "then uploads the batch of chunks to the server. This helps reduce memory pressure on " +
+        "the client when dealing with large local relations because the client does not have to " +
+        "materialize all chunks in memory.")
+      .version("4.1.0")
+      .longConf
+      .checkValue(_ > 0, "The batch size in bytes must be positive")
```

Comment:
We should check if this value is greater than the chunk size value value as an initial step